sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -472,7 +366,9 @@ export class MessageSender extends LinkEntity {
         await this._ensureTokenRenewal();
       }
     } catch (err) {
-      err = translate(err);
+      // Setting this as translate utility specifically sets to `false` if non-retryable
+      err.retryable = true;
+      err = translate(err.innerError || err);
```

Comment:
So does this change mean any error while creating the connection/sender link will now be retryable?